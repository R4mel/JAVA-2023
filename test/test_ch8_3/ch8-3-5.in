1
14
2023 12 31 23 59 59
17
6
13
14
0
10
5
6
0
0
0
