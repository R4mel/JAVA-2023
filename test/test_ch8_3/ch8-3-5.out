
************* Main Menu **************
* 0.exit 1.PersonManager 2.ch2 3.ch3 *
* 4.ch5 5.PMbyMap 6.MyVectorTest     *
**************************************
menu item? 1
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 14
date and time(ex: 2021 10 1 18 24 30)? 2023 12 31 23 59 59

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 17

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 6
-----------------
[data] directory: 0 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 13
[0] persons.txt          F 2023-11-15 07:13       327
-----------------
[data] directory: 1 files

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 14
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
-----------------
6 entries

====================== File Management Menu =====================
= 0.exit 1.fileList 2.delete 3.rename 4.addFiles 5.addDir       =
= 6.deleteAll 7.show 8.copy 9.append 10.display 11.clear 12.add =
= 13.saveText   14.loadText   15.saveTextAs   16.loadTextFrom   =
=================================================================
menu item? 0

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 10
dispPhones(): count 6
[0] p0's Phone: Apple Phone (2023.12.31 PM 11:59:59)
[1] p1's Phone: SAMSUNG Phone (2023.12.31 PM 11:59:59)
[2] s1's Phone: SAMSUNG Phone (2023.12.31 PM 11:59:59)
[3] s2's Phone: Apple Phone (2023.12.31 PM 11:59:59)
[4] w1's Phone: SAMSUNG Phone (2023.12.31 PM 11:59:59)
[5] w2's Phone: Apple Phone (2023.12.31 PM 11:59:59)

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 5
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
[6] pp0 30 71.0 true :Gwangju ,Nam-gu , Bongseon-dong 21:
[7] pp1 31 62.1 false :Jong-ro 1-gil,Jongno-gu,   Seoul:
[8] ss1 21 66.4 false :Jongno-gu Seoul: Physics 3.8 1
[9] ss2 22 55.3 true :Yeonje-gu Busan: Electronics 2.5 4
[10] ww1 23 34.3 true :Kangnam-gu Seoul: Samsung Director
[11] ww2 24 45.4 false :Dobong-gu Kwangju: Hyundai Manager

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 6
number of persons to append? 0
input 0 [delimiter(P,S,or W)] [person information] to append:
[0] p0 10 70.0 false :Gwangju ,Nam-gu , Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil,Jongno-gu,   Seoul:
[2] s1 1 65.4 true :Jongno-gu Seoul: Physics 3.8 1
[3] s2 2 54.3 false :Yeonje-gu Busan: Electronics 2.5 4
[4] w1 3 33.3 false :Kangnam-gu Seoul: Samsung Director
[5] w2 4 44.4 true :Dobong-gu Kwangju: Hyundai Manager
[6] pp0 30 71.0 true :Gwangju ,Nam-gu , Bongseon-dong 21:
[7] pp1 31 62.1 false :Jong-ro 1-gil,Jongno-gu,   Seoul:
[8] ss1 21 66.4 false :Jongno-gu Seoul: Physics 3.8 1
[9] ss2 22 55.3 true :Yeonje-gu Busan: Electronics 2.5 4
[10] ww1 23 34.3 true :Kangnam-gu Seoul: Samsung Director
[11] ww2 24 45.4 false :Dobong-gu Kwangju: Hyundai Manager

=================== Person Management Menu =====================
= 0.exit 1.display 2.clear 3.reset 4.remove 5.copy 6.append    =
= 7.insert 8.login 9.dispStudent(5_3) 10.dispPhone(5_3)        =
= 11.find(6_1) 12.wrapper(6_1) 13.shuffle(6_1) 14.setDate(6_1) =
= 15.chAddress(6_2) 16.collections(7_1) 17.fileManager(8_1)    =
================================================================
menu item? 0

************* Main Menu **************
* 0.exit 1.PersonManager 2.ch2 3.ch3 *
* 4.ch5 5.PMbyMap 6.MyVectorTest     *
**************************************
menu item? 0

Good bye!!
